staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -1080,19 +1080,18 @@ func printConfigMapVolumeSource(configMap *corev1.ConfigMapVolumeSource, w Prefi
 
 func printProjectedVolumeSource(projected *corev1.ProjectedVolumeSource, w PrefixWriter) {
 	w.Write(LEVEL_2, "Type:\tProjected (a volume that contains injected data from multiple sources)\n")
+	w.Write(LEVEL_2, "Sources:\n")
 	for _, source := range projected.Sources {
 		if source.Secret != nil {
-			w.Write(LEVEL_2, "SecretName:\t%v\n"+
-				"    SecretOptionalName:\t%v\n",
-				source.Secret.Name, source.Secret.Optional)
+			optional := source.Secret.Optional != nil && *source.Secret.Optional
+			w.Write(LEVEL_3, "SecretName:\t%v (Optional:\t%v)\n", source.Secret.Name, optional)
```

Comment:
I'd recommend focusing only fixing the bug without modifying the message or changing the `LEVEL_2`.

Reply:
No problem, so do you think the output should look like so?
```
SecretName: xxx
Optional: true/false
ConfigMap: xxx
Optional: true/false
```

Reply:
I think, it is better to stick the output that whatever currently.

Reply:
done